triton-vm/src/table.rs

```diff
@@ -32,7 +30,6 @@ pub mod program_table;
 pub mod ram_table;
 pub mod table_column;
 #[rustfmt::skip]
-pub mod tasm_air_constraints;
```

Comment:
The `rustfmt::skip` should be moved along with `tasm_air_constraints`. Now, it applies to `u32_table`, which is probably unintended.